topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	fmt.Println(ReverseString("Hello World"))
+}
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of codePoints.
+	codePoints := []rune(str)
+
+	// Here we create int that will be a pointer to the front of the codePoints.
+	var fontCodePoint int
+
+	// Here  we create int that will be a pointer to the end of the codePoints.
+	backCodePoint := len(codePoints) - 1
+
+	// While fontCodePoint is less than backCodePoint.
```

Comment:
Let’s fix these comments. Something like. 

// While there are still code points to check. 

// Swap the code points by ....

I think the increment lines don’t need comments.

Reply:
Done!